src/Components/Popover/Base/Popover.tsx

```diff
@@ -88,6 +92,28 @@ export const PopoverRoot = forwardRef<PopoverRef, PopoverProps>(
     ref
   ) => {
     const [expanded, setExpanded] = useState<boolean>(!!visible)
+    const [portalID, setPortalID] = useState<string>('')
+
+    useEffect((): (() => void) => {
+      const newPortalID = `cf-${popoverPortalName}-portal-${uuid.v4()}`
+      !portalID && setPortalID(newPortalID)
```

Comment:
Since this effect only happens once, is this check actually doing anything?

Reply:
I suppose it might be excessive, I'll test it w/o the check and see what happens